### ISSUE TITLE
Show current chapter

### ISSUE THREAD

**Issue by FichteFoll** [darsain/uosc#19]
I've had the desire to know the title of the current chapter for a while now, but I never figured out a good solution to that. In the past, I used the stats rendering (<kbd>i</kbd>), but that is hard to parse at a glance. Now I use the uosc's chapters menu, which is a *significant* improvement, but maybe it would make sense to add the current chapter to the title display?

I could see the media title and the chapter name be separated by a sequence like ` | ` or a newline, depending on how #15 develops, i.e. whether showing a multi-line title could even be considered. If not, there is the question of whether the separator is clear enough from a UX perspective or whether it could have been part of the title as well.

**Comment by darsain** [comment]
This sounds like too much clutter. I also think that using chapters menu is a better solution for this as you can toggle it with a key instead of having to drag the cursor to show the title bar.

**Comment by FichteFoll** [comment]
Last night I was thinking showing it in a second line but half the font size could work, but that might still not be simple enough (for your standards).

For one of my particular use cases, which is having a longish video open on a different monitor and just checking which chapter (song) the video is at currently, a mouse flick would be quicker than flicking the mouse and pressing a key, but I think #21 combined with just leaving the chapter menu open would solve this in a simple yet effective way.

**Comment by FichteFoll** [comment]
Just as an additional reference, YouTube added this feature a few weeks ago where they showed chapter markers (parsed from the description) in the progress bar as well as the current chapter below that next to the player control buttons. I know that usoc doesn't have that currently and probably won't get, but I wanted to provide that information regardless.

Here's an example video: https://www.youtube.com/watch?v=4g-8ZY-5ku4
